src/spotbugs/spotbugs-excludes.xml

```diff
@@ -702,10 +671,6 @@
           <Class name="jenkins.security.apitoken.LegacyApiTokenAdministrativeMonitor$RevokeAllSelectedUserAndUuid"/>
         </Or>
       </And>
-      <And>
-        <Bug pattern="VO_VOLATILE_INCREMENT"/>
-        <Class name="hudson.util.RingBufferLogHandler"/>
-      </And>
```

Comment:
Unrelated right? also, #6018

Reply:
Yeah, I took care of this in another PR and after merging it's gone from the diff.